google/cloud/spanner/internal/transaction_impl.h

```diff
@@ -49,7 +49,8 @@ class TransactionImpl {
   // passed TransactionSelector in its Client::Read()/Client::ExecuteSql()
   // call. If initially selector.has_begin(), and the operation successfully
   // allocates a transaction ID, then the functor should selector.set_id(id).
-  // Otherwise the functor should not modify the selector.
+  // Otherwise the functor should not modify the selector. A monotonically-
+  // increasing sequence number is also passed to the functor.
   template <typename Functor>
```

Comment:
Would this be a good place to use `std::enable_if<google::cloud::is_invocable<Functor, ...>>` and that acted the documentation for what are the parameters for `Functor`?

Reply:
Yes.  A new requirement since the move to Functor, but done.  Thanks.